setup.md

```diff
@@ -176,16 +176,11 @@ iOS devices.
    $ brew update
    $ brew install --HEAD libimobiledevice
    $ brew install ideviceinstaller ios-deploy
+   $ brew install cocoapods
+   $ brew link cocoapods
```

Comment:
Also feel free to merge this up onto the previous line; so just:
`brew install ideviceinstaller ios-deploy cocoapods`

The reason for splitting out libimobiledevice is because it's currently required to be installed from `HEAD`, unlike the ones on the second line.